src/VuePhoneNumberInput/index.vue

```diff
@@ -117,7 +120,10 @@
         const countryAvailable = isCountryAvailable(locale)
 
         if (countryAvailable && locale) {
-          this.countryCode = locale
+          this.countryCode = locale;
+          this.countryName = this.defaultCountryName || countries.find(function(element) {
```

Comment:
We're doing 2 passes through the countries list - once in isCountryAvailable(), and another in the .find(). I'd suggest changing isCountryAvailable() to findCountry() and use the returned value instead

Reply:
Nice suggestion, Ideally I have to do it like that. I was not preferring not to disturb the main functionality. I am completely agree on your suggestion so I will implement the same.

Thanks
Ashish